raiden-dapp/src/plugins/service-worker-assistant-plugin.ts

```diff
@@ -0,0 +1,20 @@
+import path from 'path';
+import Vue from 'vue';
+
+const SERVICE_WORKER_SCRIPT = path.join(process.env.BASE_URL ?? '/', 'service-worker.js');
+
+/**
+ * @param _Vue - global Vue instance to act on
+ * @param _options - eventual configuration for the plugin (ignored)
+ */
+export async function ServiceWorkerAssistantPlugin(
+  _Vue: typeof Vue,
+  _options?: null,
+): Promise<void> {
+  const serviceWorkerAreSupported = 'serviceWorker' in navigator;
+  const serviceWorkerShouldBeRegistered = process.env.NODE_ENV === 'production';
```

Comment:
I commented on a similar thing in another commit already. Can we somehow centralize this condition, so we have one place in the codebase for this?

Reply:
It is the same line of code...